### PR TITLE
Replace MailPoet\Models\ScheduledTask::touchAllByIds with a doctrine code [MAILPOET-4309]

### PR DESCRIPTION
**Pull request by lysyjan**
[MAILPOET-4309]

[MAILPOET-4309]: https://mailpoet.atlassian.net/browse/MAILPOET-4309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ public function process($timer = false) {
       $this->startProgress($task);
 
       try {
-        ScheduledTaskModel::touchAllByIds([$queue->taskId]);
```
Comment (alex-mailpoet) [inline mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php]: After this change, `ScheduledTaskModel` is only used for constants, can be replaced with `ScheduledTaskEntity`.

File: mailpoet/tests/integration/Newsletter/Sending/ScheduledTasksRepositoryTest.php [hunk 2]
```diff
@@ -63,6 +69,28 @@ public function testItCanGetFutureScheduledTasks() {
     $this->assertSame($expectedResult, $tasks);
   }
 
+  public function testItTouchAllScheduledTasksByIds(): void {
```
Comment (alex-mailpoet) [inline mailpoet/tests/integration/Newsletter/Sending/ScheduledTasksRepositoryTest.php]: Grammar: `Touch` -> `Touches`.

File: mailpoet/tests/integration/Newsletter/Sending/ScheduledTasksRepositoryTest.php [hunk 3]
```diff
@@ -63,6 +69,28 @@ public function testItCanGetFutureScheduledTasks() {
...
+    $this->repository->touchAllByIds([$touched[0]->getId()]);
+
+    // check touched task
+    $now = CarbonImmutable::createFromTimestamp((int)$this->wp->currentTime('timestamp'));
+    $this->entityManager->clear();
+    $scheduledTask = $this->repository->findOneById($touched[0]->getId());
+    $this->assertInstanceOf(ScheduledTaskEntity::class, $scheduledTask);
+    $updatedAt = $scheduledTask->getUpdatedAt();
+    $this->assertInstanceOf(\DateTime::class, $updatedAt);
+    $this->assertEqualsWithDelta($updatedAt->getTimestamp(), $now->getTimestamp(), 1);
```
Comment (alex-mailpoet) [inline mailpoet/tests/integration/Newsletter/Sending/ScheduledTasksRepositoryTest.php]: I see potential flakiness here. If code execution takes more than 1 second, the test will fail. I think a less strict comparison would work just fine: 
`$this->assertGreaterThan($updatedAt->getTimestamp(), $originalUpdatedAt->getTimestamp());`.

**Review by alex-mailpoet — DISMISSED** [review]
Thank you for the changes! All is good!

**Review by veljkho — DISMISSED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**